src/Atmos/Model/moisture.jl

```diff
@@ -0,0 +1,41 @@
+#### Moisture component in atmosphere model
+abstract type MoistureModel end 
+
+struct DryModel <: MoistureModel
+end
+vars_state(::DryModel) = ()
+function flux!(m::DryModel, flux::Grad, state::State, diffusive::State, auxstate::State, t::Real)
+end
+function preodefun_elem!(m::DryModel, auxstate::State, state::State, t::Real)
+end
+
+function pressure(::DryModel, state, diffusive, auxstate, t)
+  T = eltype(state)
+  γ = T(γ_exact)
+  ρinv = 1 / state.ρ
+  P = (γ-1)*(state.ρe - ρinv * (state.ρu^2 + state.ρv^2 + state.ρw^2) / 2)
```

Comment:
use MoistThermo code

Reply:
We do not need a pressure function at all here. Just use `air_pressure`.